source/transfer_list.rst

```diff
@@ -820,6 +821,48 @@ and reserved for use as a TPM Command Response Buffer interface.
      - Size of CRB.
 
 
+.. _rw_mem_layout:
+
+Read-Write Memory Layout Entry Layout (XFERLIST_RW_MEM_LAYOUT)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This type defines the layout of a region of read-write memory that a client can
+allocate into its memory map. If other memory types are required (i.e.
+read-only memory), separate TE's should be defined.
```

Comment:
This sounds very specific to a single project (TF-A?), so it should probably go in the TE section for that. At least I wouldn't know from this description what to do with this entry if I encountered it in my firmware.

Reply:
You're probably right. The structure this TE represents is fairly simple:
``` c
typedef struct meminfo {
	uintptr_t total_base;
	size_t total_size;
} meminfo_t;
```
This to me seems adaptable to other contexts, although i'm yet to find any concrete examples. In any case, I've moved it to TF-A's namespace. If a more general use case materialises then we can consider adding it back.